ui/src/utils/PromptApi.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: merge with UtteranceApiService.js (in Ashley's PR)
+
+class PromptApi {
```

Comment:
Is there a reason this is a class? It doesn't look like getNewPrompt or resetAllPromptsUnread shares any state. Could the functions be exported separately?
```
export getNewPrompt
export resetAllPromptsUnread
```

And then * imported into an object:
```
import * as promptsApi from ...
```